packages/framework/autoload-easy-coding-standard.yml

```diff
@@ -27,3 +27,21 @@ parameters:
         PHP_CodeSniffer\Standards\Generic\Sniffs\CodeAnalysis\EmptyStatementSniff.DetectedWhile:
             - '*/src/Model/Product/Availability/ProductAvailabilityRecalculator.php'
             - '*/src/Model/Product/Pricing/ProductPriceRecalculator.php'
+
+        Shopsys\CodingStandards\Sniffs\ObjectIsCreatedByFactorySniff:
+          - '*/tests/*'
+          - '*/src/DataFixtures/Base/CategoryRootDataFixture.php'
+          - '*/src/DataFixtures/Demo/ArticleDataFixture.php'
+          - '*/src/DataFixtures/Demo/CategoryDataFixture.php'
+          - '*/src/DataFixtures/Demo/PaymentDataFixture.php'
+          - '*/src/DataFixtures/Demo/PersonalDataAccessRequestDataFixture.php'
+          - '*/src/DataFixtures/Demo/TransportDataFixture.php'
+          - '*/src/DataFixtures/DemoMultidomain/ArticleDataFixture.php'
```

Comment:
Here it would be easy to maintain to use `fnmatch()`:

```diff
       Shopsys\CodingStandards\Sniffs\ObjectIsCreatedByFactorySniff:
-          - '*/tests/*'
-          - '*/src/DataFixtures/Base/CategoryRootDataFixture.php'
-          - '*/src/DataFixtures/Demo/ArticleDataFixture.php'
-          - '*/src/DataFixtures/Demo/CategoryDataFixture.php'
-          - '*/src/DataFixtures/Demo/PaymentDataFixture.php'
-          - '*/src/DataFixtures/Demo/PersonalDataAccessRequestDataFixture.php'
-          - '*/src/DataFixtures/Demo/TransportDataFixture.php'
-          - '*/src/DataFixtures/DemoMultidomain/ArticleDataFixture.php'
+          - '*/src/DataFixtures/*/*Fixture.php'
```

Reply:
But this would ignore all data fixtures. Only these seven (out of more than fifty fixtures) are violating the sniff.

Reply:
I see. Why not fix it then if it's only just few files?

These ignores (in PHPStan, Code sniffers, ECS) usually suggest it's instentional in whole group. Like in FormTypes. 

Comment above would also help.

Reply:
They should be fixed in  #252, we wanted the sniff in master as soon as possible.

Reply:
I see. 

That would be great to have here in PR description. I could click on that instead of writing comments, /cc @boris-brtan